chapter11/ajd/exercise2.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use IslandDate;
+
+my ( $mday, $mon, $year, $wday ) = ( localtime )[ 3 .. 6 ];
+
+$wday = IslandDate::number_to_day_name( $wday );
+$mon = IslandDate::number_to_month_name( $mon );
+$year += 1900;
```

Comment:
I know, this was mostly just an extra to show off Time::Piece a bit. Also because I used ``use``, this was invoked at compile time, so the ``localtime`` above is Time::Piece's. It preserves the list and stringified context functionality of ``localtime``, while also returning an object in scalar context.